rocky/rocky/views/task_detail.py

```diff
@@ -0,0 +1,79 @@
+import uuid
+
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+from django.views.generic import TemplateView
+from katalogus.views.mixins import BoefjeMixin, NormalizerMixin
+
+from rocky.scheduler import client
+from rocky.views.mixins import OctopoesView
+
+
+class TaskDetailView(OctopoesView, TemplateView):
+    @staticmethod
+    def get_task(task_id):
+        return client.get_task_details(task_id)
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        context["task_id"] = uuid.UUID(kwargs["task_id"]).hex
+        context["task"] = self.get_task(context["task_id"])
+        return context
+
+
+class BoefjeTaskDetailView(BoefjeMixin, TaskDetailView):
+    template_name = "tasks/boefje_task_detail.html"
+    plugin_type = "boefje"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        context["breadcrumbs"] = [
+            {
+                "url": reverse("task_list", kwargs={"organization_code": self.organization.code}),
+                "text": _("Tasks"),
+            },
+            {
+                "url": reverse(
+                    "boefje_task_view",
+                    kwargs={"organization_code": self.organization.code, "task_id": uuid.UUID(context["task"].id).hex},
```

Comment:
The key `task_id` already exists in the super's context, so you can use that instead of recreating a UUID object

Reply:
Obviously. Thanks for pointing this out.